targets/HUDL/main.cpp

```diff
@@ -21,12 +21,12 @@ namespace time = EVT::core::time;
 namespace log = EVT::core::log;
 using namespace std;
 
-const uint32_t SPI_SPEED = SPI_SPEED_500KHZ;
-const uint8_t deviceCount = 1;
+#define SPI_SPEED SPI_SPEED_500KHZ
+#define DEVICE_COUNT 1
 
 ///////////////////////////////////////////////////////////////////////////////
-// EVT-core CAN callback and CAN setup. This will include logic to set
-// aside CANopen messages into a specific queue
+// EVT-core CAN call back and CAN set up. This will include logic to set     //
+// aside CANOpen messages into a specific queue                              //
```

Comment:
[CANopen](https://en.wikipedia.org/wiki/CANopen)
```suggestion
// aside CANopen messages into a specific queue                              //
```